app/models/role.rb

```diff
@@ -50,20 +51,24 @@ def self.columns
 
   before_destroy :prevent_destruction_unless_destroyable
   after_update :touch_role_appointments
-  after_save :republish_organisation_to_publishing_api
+  after_save :republish_organisations_to_publishing_api
 
   extend FriendlyId
   friendly_id
 
   include TranslatableModel
   translates :name, :responsibilities
 
-  def republish_organisation_to_publishing_api
+  def republish_organisations_to_publishing_api
     organisations.each do |organisation|
-      Whitehall::PublishingApi.republish_async(organisation)
+      republish_organisation_to_publishing_api(organisation)
     end
   end
 
+  def republish_organisation_to_publishing_api(organisation)
+    Whitehall::PublishingApi.republish_async(organisation)
```

Comment:
`.bulk_republish_async` would better replicate how the Publishing API would prioritise this, if it was being done "properly" through dependency resolution. It doesn't really matter though, this is fine as well I think.